components/profile/Cargo.toml

```diff
@@ -25,4 +25,4 @@ regex = { workspace = true }
 
 [target.'cfg(not(target_os = "windows"))'.dependencies]
 libc = { workspace = true }
-servo_allocator = { path = "../allocator" }
```

Comment:
Why?

Reply:
In this case the code is trying to get stats from jemalloc, so it didn't seem to make sense to abstract away the interface into `servo_allocator` and instead get the stats directly from jemalloc itself.

Reply:
But if we ever (and we will likely try) to replace allocator it's easier to have things abstracted.

Reply:
It feels like in this case, the allocator should expose a higher-level function, since `mallctl` is jemalloc specific. Perhaps we can remove the abstraction now and correct it later when we add an alternative allocator?

Reply:
Sounds good!